sdk/tests/tutorials/properties/test_derived_properties.py

```diff
@@ -0,0 +1,132 @@
+import unittest
+
+import json
+
+import lusid
+
+import lusid.models as models
+from utilities import InstrumentLoader
+from utilities import TestDataUtilities
+
+class DerivedPropertyTests(unittest.TestCase):
+
+
+    @classmethod
+    def setUpClass(cls):
+        # create a configured API client
+        api_client = TestDataUtilities.api_client()
+        cls.scopes_api = lusid.ScopesApi(api_client)
+        cls.property_definitions_api = lusid.PropertyDefinitionsApi(api_client)
+        cls.instruments_api = lusid.InstrumentsApi(api_client)
+        # load instruments from InstrumentLoader
+        instrument_loader = InstrumentLoader(cls.instruments_api)
+        cls.instrument_ids = instrument_loader.load_instruments()
+
+    def create_ratings_property(self, figi, fitch_value=None, moodys_value=None):
+
+        ratings = ["Fitch", "Moodys"]
+
+        for rating in ratings:
+            property_definition = models.CreatePropertyDefinitionRequest(
+                domain="Instrument",
+                scope="Test-Demo",
+                code="{}Rating".format(rating),
+                display_name="{}Rating".format(rating),
+                data_type_id=lusid.ResourceId(
+                    scope="system", code="number"))
+
+            try:
+                # create property definition
+                self.property_definitions_api.create_property_definition(
+                    create_property_definition_request=property_definition)
+            except lusid.ApiException as e:
+                if json.loads(e.body)["name"] == "PropertyAlreadyExists":
+                    pass
+
+        properties = {"Instrument/Test-Demo/FitchRating": fitch_value, "Instrument/Test-Demo/MoodysRating": moodys_value}
+
+        # upsert property definition
+        for key in properties:
+            if properties[key] is not None:
+                property_request = [
+                    models.UpsertInstrumentPropertyRequest(
+                        identifier_type="Figi",
+                        identifier=figi,
+                        properties=[
+                            models.ModelProperty(key=key,
+                                                 value=models.PropertyValue(
+                                                     metric_value=models.MetricValue(value=properties[key]))
+                                                 )]
+                    )]
+
+                response = self.instruments_api.upsert_instruments_properties(
+                    upsert_instrument_property_request=property_request)
+
+    def get_instruments_with_deriv_prop(self, Figi):
+        response = self.instruments_api.list_instruments(
```

Comment:
Why can't we use get_instrument() which is much more efficient when we know the unique IDs of the instrument we want?

Reply:
See response when using get_instrument() call -- would appear that it doesn't find the properties with the given Figi, returns an empty list. Not sure if there is another way to call the properties themselves (i.e. get_property)). 

>>> response: properties[]

```
'values': {'BBG000FD8G46': {'href': 'https://swagpotato.lusid.com/api/api/instruments/LusidInstrumentId/LUID_L6GFAZUE',
                             'identifiers': {'Figi': 'BBG000FD8G46',
                                             'LusidInstrumentId': 'LUID_L6GFAZUE'},
                             'instrument_definition': None,
                             'links': [{'description': None,
                                        'href': 'https://swagpotato.lusid.com/api/api/schemas/entities/Instrument',
                                        'method': 'GET',
                                        'relation': 'EntitySchema'}],
                             'lookthrough_portfolio': None,
                             'lusid_instrument_id': 'LUID_L6GFAZUE',
                             'name': 'HISCOX LTD',
                             'properties': [],
                             'state': 'Active',
                             'version': {'as_at_date': datetime.datetime(2020, 9, 30, 9, 19, 39, 9127, tzinfo=tzutc()),
                                         'effective_from': datetime.datetime(1, 1, 1, 0, 0, tzinfo=tzutc())}}}}
```

Reply:
this doesn't seem right, are you saying that derived properties don't come back?

Reply:
Can you share your get_instrument request?
You need to pass it the property_keys you want to see.
Example:

        looked_up_instruments = self.instruments_api.get_instruments(identifier_type="Figi",
                                                                     request_body=[figi],
                                                                     property_keys=[
                                                                         "Instrument/default/ClientInternal"
                                                                     ])

Reply:
That resolved it, needed the correct property keys for the derived scope. Adjusted with get_instrument for latest push.